recipe/build_base.bat

```diff
@@ -1,17 +1,30 @@
-set LIB=%LIBRARY_LIB%;%LIB%
-set LIBPATH=%LIBRARY_LIB%;%LIBPATH%
-set INCLUDE=%LIBRARY_INC%;%INCLUDE%
+@echo on
 
-ECHO [directories] > mplsetup.cfg
-ECHO basedirlist = %LIBRARY_PREFIX% >> mplsetup.cfg
-ECHO [packages] >> mplsetup.cfg
-ECHO tests = False >> mplsetup.cfg
-ECHO sample_data = False >> mplsetup.cfg
-ECHO toolkits_tests = False >> mplsetup.cfg
-ECHO [libs] >> mplsetup.cfg
-ECHO system_freetype = True >> mplsetup.cfg
+set "MESON_ARGS=%MESON_ARGS% --buildtype=release --prefix=%LIBRARY_PREFIX% --pkg-config-path=%LIBRARY_LIB%\pkgconfig -Dlibdir=lib -Dsystem-freetype=true -Dsystem-qhull=true"
 
-set MPLSETUPCFG=mplsetup.cfg
+if "%CI%" == "azure" (
+    :: Hack to try removing problematic Python from Azure CI image
+    set "CLEANUP_DIRS=C:\hostedtoolcache\windows\Python;"
+    mkdir C:\empty
+    for %%f in (%CLEANUP_DIRS:;= %) do (
+        echo Removing %%f
+        robocopy /mir /ns /nc /nfl /ndl /np /njh /njs %%f > nul 2>&1
+        rmdir /q %%f
+        if errorlevel 1 (
+            echo Failed to remove
+            rmdir /q C:\empty
+            exit /b 1
+        )
+    )
+    rmdir /q C:\empty
+)
```

Comment:
Am pulling out this cleanup logic into conda-smithy: https://github.com/conda-forge/conda-smithy/pull/1966

Think it will better serve us there and we can clean this up after